### PR TITLE
Add API Spec to README.md

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,109 @@
...
+
+## API  Spec
+
+### Methods
+
+ - **/endpoint/**
+    - **GET** - List all objects. Will be called **LIST** in further mentions
+    - **POST** - Create an object. Will be called **POST** in further mentions
+ - **/endpoint/\<obj_id\>/**
+    - **GET** - Get exact object data. Will be called **GET** in further mentions
```
Comment (madlabman) [inline README.md:13]: What about `delete` method?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,109 @@
...
+cluster | `integer` | `True` | FK to [Cluster](#cluster)
+filesystem | `integer` | `True` | FK to [File system](#file-system)
+
+
+### Backup
+
+Create backup of cluster to file system.
+
+> [Connection](#connection) for given [Cluster](#cluster) and [File system](#file-system)
+> should exist before backup creation
```
Comment (madlabman) [inline README.md:90]: So it's better to pass connection id, isn't it?

File: README.md (hunk 1 above)
Comment (dgusakov) [reply README.md:13]: This will be a demo app. So it looks reasonable to keep it small but still representative.

File: README.md (hunk 2 above)
Comment (dgusakov) [reply README.md:90]: Maybe. But this is an illustration of possible indirect dependency. So we need it to be so. In real apps such dependencies can be a result of a more complex data model or business logic.

**Pull request merged**